examples/plotting/wcsaxes_plotting_example.py

```diff
@@ -0,0 +1,54 @@
+"""
+===========================
+WCS Axes Plotting
+===========================
+In this example we are going to look at how pixel and map coodinates work with WCS Axes.
+"""
+
+import matplotlib.pyplot as plt
+
+from astropy import units as u
+
+import sunpy.data.sample
+import sunpy.map
+
+###############################################################################
+# Lets start by creating a Map from the sample data.
+my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
+
+###############################################################################
+# Lets choose xlimits and ylimits in arcsecs that we are interested in.
+xlims_world = [500, 1100]*u.arcsec
+ylims_world = [-800, 0]*u.arcsec
```

Comment:
What's the purpose of these variables?  They don't seem to be used.

Reply:
Hey @ayshih actually it was mentioned in the issue to add xlim and ylim, but yeah they haven't been used later in the code. so will remove it.